vignettes/O4_make_a_package.Rmd

```diff
@@ -24,7 +24,7 @@ Saving your data and analyses into an R package helps to create reproducible res
 ## How do I make a package project?
 All you need to do is add `vignette = TRUE` when you use the `make_project()` function. Once the project is created, run the **run_me.R** file.  For example type:
 
-1. `make_project("~/Desktop/my_example", "R", example=TRUE, vignette=TRUE)`.  This creates a new project with all the files for a package in the current directory.  The vignette includes an example table and figure and all the R Markdown syntax for adding hyperlinks to the table and figures.
+1. `rUM::make_project("~/Desktop/my.example", "R", example=TRUE, vignette=TRUE)`.  This creates a new project with all the files for a package in the current directory.  The vignette includes an example table and figure and all the R Markdown syntax for adding hyperlinks to the table and figures.
```

Comment:
any particular reason for using the `.` and not `_`?
otherwise all good

Reply:
A package name can not include an _